### PR TITLE
#1-4Task max.kalinin

### PR DESCRIPTION
**Pull request by MaxKalinin92**
https://travis-ci.org/github/MaxKalinin92/js-assignments

### CODE REVIEW

**Comment by AlexMokin** [comment]
fix formatting. There shouldn't be any changes to formatting of exsting code. E.g. here
image

I'll mark the PR as "Draft", please click "ready for review" when you finish fixes. Thank you!

File: task/02-numbers-tasks.js [hunk 1]
```diff
@@ -202,9 +204,15 @@ function roundToPowerOfTen(num, pow) {
  *   17 => true
  */
 function isPrime(n) {
-    throw new Error('Not implemented');
+    for (let i = 2; i <= Math.sqrt(n); i++) {
+        if (n % i == 0) {
```
Comment (vramaniuk) [inline task/02-numbers-tasks.js]: "operator === is always preferable
sometimes ''==' may cause an error
(check at console :
""0"" == 0
and ""0"" === 0)"

**Review by vramaniuk — COMMENTED** [review]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by vramaniuk** [comment]
> https://travis-ci.org/github/MaxKalinin92/js-assignments

Please, privide the direct link to your task success travis build https://travis-ci.org/github/MaxKalinin92/js-assignments/builds/706638560

**Comment by MaxKalinin92** [comment]
https://travis-ci.org/github/MaxKalinin92/js-assignments/builds/721400492

**Comment by vramaniuk** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**